runtime/moonbase/src/lib.rs

```diff
@@ -1485,6 +1496,7 @@ construct_runtime! {
 		EvmForeignAssets: pallet_moonbeam_foreign_assets::{Pallet, Call, Storage, Event<T>} = 56,
 		Parameters: pallet_parameters = 57,
 		XcmWeightTrader: pallet_xcm_weight_trader::{Pallet, Call, Storage, Event<T>} = 58,
+		WeightReclaim: cumulus_pallet_weight_reclaim = 59,
```

Comment:
From now on, lets try to have the same pellet index on all runtimes. This makes things easier when sharing encoded payloads.

We have `WeightReclaim: cumulus_pallet_weight_reclaim = 118` in moonbeam and moonriver.

This pallet does not expose any extrinsic, but it is still a good practice to have.